packages/network/src/api/image/requests.ts

```diff
@@ -63,6 +63,10 @@ export interface AddBeautyShotImageOptions {
    * The list of tasks to run for this image.
    */
   tasks: TaskName[];
+  /**
+   * Boolean indicating if a thumbnail request will be send when addImage is called.
+   */
+  enableThumbnail: boolean;
```

Comment:
_...will be sen**t** when..._

Also : rename variable to `useThumbnailCaching`, set it to optional with default : `true`